BlueprintCore/BlueprintCore.csproj

```diff
@@ -94,6 +94,14 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    
+   <PackageReference Include="System.Text.Json" Version="6.0.1" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <None Update="resources\encyclopedia.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
You might need to specify as content to work properly w/ NuGet. When you build the NuGet package is created--it's just a zip file so you can open and make sure the content is there.

Still you should verify this works w/ installing the NuGet package as well. It's unfortunately annoying but [here are the steps to install locally through VS](https://stackoverflow.com/questions/10240029/how-do-i-install-a-nuget-package-nupkg-file-locally).[]

Reply:
I resorted to adding content as EmbeddedResource to make it work. I tested that it works by creating a local project, adding a local nuget package and running a test.

I suppose it could be possible to make tests load WW-Blueprint-Core from local nuget package, but it sounds like the pain in the ass. I don't think it's necessary to do right now, do you?